docs/pin.rst

```diff
@@ -23,57 +24,65 @@ depending upon the digital reading on pin 0::
-|  10 | Analog  | Column 3 |
-+-----+---------+----------+
-|  11 | Digital | Button B |
-+-----+---------+----------+
-|  12 | Digital |          |
-+-----+---------+----------+
-|  13 | Digital | SPI SCK  |
-+-----+---------+----------+
-|  14 | Digital | SPI MISO |
-+-----+---------+----------+
-|  15 | Digital | SPI MOSI |
-+-----+---------+----------+
-|  16 | Digital |          |
-+-----+---------+----------+
-+-----+---------+----------+
-|  19 | Digital | I2C SCL  |
-+-----+---------+----------+
-|  20 | Digital | I2C SDA  |
-+-----+---------+----------+
++-----+---------+----------+-----------------+
+| Pin | Type    | Function | Function **V2** |
++=====+=========+==========+=================+
+|  0  | Touch   | Pad 0    | Pad 0           |
++-----+---------+----------+-----------------+
+|  1  | Touch   | Pad 1    | Pad 1           |
++-----+---------+----------+-----------------+
+|  2  | Touch   | Pad 2    | Pad 2           |
++-----+---------+----------+-----------------+
+|  3  | Analog  | Column 1 | Column 3        |
++-----+---------+----------+-----------------+
+|  4  | Analog  | Column 2 | Column 1        |
++-----+---------+----------+-----------------+
+|  5  | Digital | Button A | Button A        |
++-----+---------+----------+-----------------+
+|  6  | Digital | Column 9 | Column 4        |
++-----+---------+----------+-----------------+
+|  7  | Digital | Column 8 | Column 2        |
++-----+---------+----------+-----------------+
+|  8  | Digital |          |                 |
++-----+---------+----------+-----------------+
+|  9  | Digital | Column 7 |                 |
++-----+---------+----------+-----------------+
+|  10 | Analog  | Column 3 | Column 5        |
++-----+---------+----------+-----------------+
+|  11 | Digital | Button B | Button B        |
++-----+---------+----------+-----------------+
+|  12 | Digital |          |                 |
++-----+---------+----------+-----------------+
+|  13 | Digital | SPI SCK  | SPI SCK         |
++-----+---------+----------+-----------------+
+|  14 | Digital | SPI MISO | SPI MISO        |
++-----+---------+----------+-----------------+
+|  15 | Digital | SPI MOSI | SPI MOSI        |
++-----+---------+----------+-----------------+
+|  16 | Digital |          |                 |
++-----+---------+----------+-----------------+
++-----+---------+----------+-----------------+
+|  19 | Digital | I2C SCL  |                 |
++-----+---------+----------+-----------------+
+|  20 | Digital | I2C SDA  |                 |
++-----+---------+----------+-----------------+
 
 The above table summarizes the pins available, their types (see below) and what
 they are internally connected to.
 
+The latest micro:bit device **V2** has two additional pins that you can access
+in MicroPython, but that are not available via the edge connector:
+
+- ``pin_logo`` - A touch sensitive logo pin on the front of the micro:bit. which by 
+default is set to capacitive touch mode.
```

Comment:
This line is not rendered correctly 
![image](https://user-images.githubusercontent.com/29712657/100263046-e7e8b000-2f44-11eb-967a-6b44148e8f3e.png)

Reply:
As this comment is in an outdated commit, I've reopened in https://github.com/bbcmicrobit/micropython/pull/692/files#r531057188